rsvp/views.py

```diff
@@ -37,14 +43,44 @@ def event_signup(request, event_id):
         raise PermissionDenied
     event = get_object_or_404(Event, id=event_id)
     rsvp = Reservation.objects.filter(event=event, user=user)
-    print("event_signup",len(rsvp))
+    emails = []
+    variables = {}
+    nameDict = {}
+    context = {}
+    context["name"] = user.first_name
+    emails.append(user.email)
```

Comment:
maybe we should always add `seoul+events@djangogirls.org` in there so we know when emails go out